app/src/lib/release-notes.ts

```diff
@@ -0,0 +1,105 @@
+const ordinal: (i: number) => string = require('ordinal')
+
+import {
+  ReleaseMetadata,
+  ReleaseNote,
+  ReleaseSummary,
+} from '../models/release-notes'
+
+const itemEntryRe = /^\[(new|fixed|improved|removed|added|pretext)\]\s(.*)/i
+
+const monthNames = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+]
+
+function formatDate(date: Date) {
+  const day = date.getDate()
+  const dayWithSuffix = ordinal(day)
+  const monthIndex = date.getMonth()
```

Comment:
a more minimal change than what was proposed above would be to replace this line with `date.toLocaleDateString('en-US', { month: 'long' });` and remove the months names list

Reply:
I'm fine with dropping the dependency on `ordinal` above